autoload/codefmt/formatterhelpers.vim

```diff
@@ -87,21 +87,32 @@ endfunction
 " the tool for range formatting and post a URL for that feature request above
 " code that calls it.
 "
-" @throws ShellError if the {cmd} system call fails
+" vararg0: if truthy, the syscall ignores errors. This can be helpful for
```

Comment:
The convention is to put the actual name in square braces (instead of calling it vararg0) and vimdoc detects it as an optional parameter.

Reply:
In the process of hunting for documentation on vimdoc, I found out that vimscript can use default values. That seems more appropriate than variable args here.

I reformatted the comment to be hopefully more idiomatic; PTAL.

Reply:
As of 8.1.1310, apparently: https://github.com/vim/vim/commit/42ae78cfff

I'm sure it's paranoia, the code was submitted in 2019, but still we tend to err on the side of supporting very old vim versions. I'm trying to cross-reference when that version would've reached e.g. Debian stable and other major distros... Would you be able to help research that for my peace of mind? 😅

Reply:
Also still need to add support for it in the vimdoc tool apparently: https://github.com/google/vimdoc/issues/114

Does it work if you explicitly add `@default ignoreerrors 0` in the doc comments, or does vimdoc just complain that it's not optional and shouldn't have a default?

Reply:
Is there a command somewhere to generate the vimdoc? Or can you explain what you mean by "does it work"? I'm new to vim development so I'm not sure what I need to test.

Reply:
OK, looking at Ubuntu LTS releases, 20.04 Focal has vim 8.1.2269 (OK), but 18.04 Bionic has vim 8.0.1453 (Not OK). Bionic is just over the five year mark; 14.04 is still getting security updates so in theory someone could be using this plugin there. I miss Live at Head from my Google days.

I went ahead and reverted to the vararg version, and gave vimdoc a @usage hint. PTAL.

Reply:
Thanks for humoring me there. 🤓

Reply:
> Is there a command somewhere to generate the vimdoc?

And yes, there's a [vimdoc](https://github.com/google/vimdoc) tool to fully generate these docs. I can just tweak and regen those on my end after we merge this for simplicity's sake.